src/main/log-handler.js

```diff
@@ -10,8 +10,8 @@ function logName () {
   }
   const fileName = [
     `${d.getFullYear()}-`,
-    `${pad(d.getMonth())}-`,
-    `${pad(d.getDay())}-`,
+    `${pad(d.getMonth() + 1)}-`,
+    `${pad(d.getDate())}-`,
```

Comment:
is getDate() correct here? Looks like it should be getDay()

Reply:
It is correct, Javascript it just stupid: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate